src/python/grpcio_tests/tests/unit/framework/interfaces/face/_future_invocation_asynchronous_event_service.py

```diff
@@ -450,6 +452,7 @@ def testFailedUnaryRequestUnaryResponse(self):
           with self.assertRaises(face.ExpirationError):
             response_future.result()
           self.assertIsNotNone(response_future.traceback())
+          self.assertTrue(abortion_callback._called)
```

Comment:
This ends up hanging.  ```add_callback()``` calls the callback with no arguments, and ```abortion_callback.future()``` loops until the callback is called with a future.